### PR TITLE
add burger menu

### PR DESCRIPTION
**Pull request by Valentyna08**

### CODE REVIEW

File: src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import logo from '../../assets/Logo.svg';
+import './Logo.scss';
+
+export const Logo: React.FC = () => {
+  return (
+    <div className="logo">
+      <a href="home">
+        <img className="logo__img" src={`${logo}`} alt="Nice gadgets logo" />
+      </a>
```
Comment (ilona-borkovska) [inline src/components/Logo/Logo.tsx]: NavLink, remove `${}`

File: src/components/BurgerMenu/BurgerMenu.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
+@import '../../styles/utils/variables';
+@import '../../styles/utils/mixins';
+
+.burger {
+  overflow-y: auto;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: space-between;
+  position: absolute;
```
Comment (ilona-borkovska) [inline src/components/BurgerMenu/BurgerMenu.scss]: check css order

File: src/components/Header/Header.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
+@import '../../styles/utils/breakpoints';
+@import '../../styles/utils/variables';
+@import '../../styles/utils/mixins';
+
+iframe {
+  display: none;
+}
```
Comment (ilona-borkovska) [inline src/components/Header/Header.scss]: remove

File: src/components/Header/Header.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  height: 48px;
+
+  border-bottom: solid 1px $color-white;
```
Comment (ilona-borkovska) [inline src/components/Header/Header.scss]: change color

File: src/components/Header/Header.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+  background-color: $color-dark;
+
+  &__left {
+    display: flex;
+    justify-content: flex-start;
```
Comment (ilona-borkovska) [inline src/components/Header/Header.scss]: why do you need that flex start? it is init value

File: src/components/BurgerMenu/BurgerMenu.scss [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+  right: 0;
+  bottom: 0;
+  background-color: $color-dark;
+
+  // transform: translateX(-150%);
+
+  width: 100vw;
+  height: 100vh;
+
+  // transition: transform $effectDuration;
```
Comment (ilona-borkovska) [inline src/components/BurgerMenu/BurgerMenu.scss]: remove comments

**Review by ilona-borkovska — APPROVED** [review]
Nice)

**Review by kochetov-yevhen-may29 — APPROVED** [review]

**Pull request merged**